designs/virtualization/4.5/Scheduling/Affinity/Affinity.md

```diff
@@ -0,0 +1,65 @@
+---
+parent: Virtualization
+version: 4.5
+---
+
+# Affinity
+
+Introducing Affinity as it appears in-page and the Edit modal.
+
+## in-page placement
+
+![Affinity field in the Scheduling section ](img/Affinity-0-0.jpg)
+The affinity field is placed in the Workload details page > Details tab > Scheduling section > Policies columns.
+
+![Affinity field in the Scheduling section ](img/Affinity-0-1.jpg)
+The affinity field counts the number of rules applied.
+
+## Affinity rules list
+
+![Affinity all rules modal ](img/Affinity-1-0.jpg)
+On clicking on the Edit pencil icon, the user will be presented with a modal, containing all the applied rules for this workload.
+Clicking on the 'Add affinity rule' button will replace this modal with the 'Add affinity rule' modal.
+Selecting 'Edit' from each existing rule kebab menu will replace this modal with the 'Edit affinity rule' modal
+
+A support text below the header will notify the user that:
+"Rules with ‘Preferred’ condition will stack with an ‘AND’ relation between them. 
+Rules with ‘Required condition will have an ‘OR’ relation between them."
+
+The "learn more about affinity rules" links to:
+https://kubevirt.io/2020/Advanced-scheduling-with-affinity-rules.html
+
+## Add/Edit affinity rule modal
+
+![Affinity Edit/Create modal ](img/Affinity-2-0.jpg)
```

Comment:
In the implementation the title for the modal is either "New affinity" or "Edit affinity" the action is "Save affinity" for both the add and edit modals.